### PR TITLE
Feature/bread crumb

### PR DESCRIPTION
**Pull request by sathish199308**
Bread Crumb Implementation

### CODE REVIEW

File: BookingSystem/Controllers/BookingController.cs [hunk 1]
```diff
@@ -43,6 +44,8 @@ public class BookingController : Controller
             _clientVenueRepository = clientVenueRepository;
             _clientRoomRepository = clientRoomRepository;
         }
+
+        [Breadcrumb("BookingList", FromAction = "Index", FromController = typeof(CustomerController))]
```
Comment (vivekr2021) [inline BookingSystem/Controllers/BookingController.cs:48]: Please do not hard-code the Action name. Use FormAction = nameof(Index) instead FormAction = "Index"

File: BookingSystem/Controllers/CustomerController.cs [hunk 2]
```diff
@@ -56,7 +61,9 @@ public async Task<IActionResult> Index()
             return View();
         }
 
+      
         [HttpPost]
+        [Breadcrumb("Advanced", FromAction = "Index", FromController = typeof(CustomerController))]
```
Comment (vivekr2021) [inline BookingSystem/Controllers/CustomerController.cs:66]: Please do not hard-code the Action name.

File: BookingSystem/Controllers/HotelController.cs [hunk 3]
```diff
@@ -45,6 +45,7 @@ public async Task<ActionResult> Create()
         }
 
         [HttpGet]
+        [Breadcrumb("HotelList", FromAction = "Index", FromController = typeof(BookingController))]
```
Comment (vivekr2021) [inline BookingSystem/Controllers/HotelController.cs:48]: Please do not hard-code the Action name.

File: BookingSystem/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -94,7 +94,9 @@
             </div>
         </nav>
     </header>
+    <breadcrumb></breadcrumb>
```
Comment (vivekr2021) [inline BookingSystem/Views/Shared/_Layout.cshtml:97]: Move the breadcrumb attribute inside the container

**Review by vivekr2021 — CHANGES_REQUESTED** [review]

File: BookingSystem/Controllers/BookingController.cs (hunk 1 above)
Comment (sathish199308) [reply BookingSystem/Controllers/BookingController.cs:48]: Hi Sir, I changed Form Action to nameofIndex

File: BookingSystem/Controllers/CustomerController.cs (hunk 2 above)
Comment (sathish199308) [reply BookingSystem/Controllers/CustomerController.cs:66]: Hi Sir, I changed Form Action to nameof(Index)

File: BookingSystem/Controllers/HotelController.cs (hunk 3 above)
Comment (sathish199308) [reply BookingSystem/Controllers/HotelController.cs:48]: Hi Sir I changed From Action to name of Index

File: BookingSystem/Views/Shared/_Layout.cshtml (hunk 4 above)
Comment (sathish199308) [reply BookingSystem/Views/Shared/_Layout.cshtml:97]: Sorry Sir, I miss to move breadcrumb to container, Right now I moved it is properly aligned

**Pull request merged**